### ISSUE TITLE
Problem with using amberoad/bert-multilingual-passage-reranking-msmarco cross encoder

### ISSUE THREAD

**Issue by sbhttchryy** [UKPLab/sentence-transformers#732]
I tried to use the  'qa_retrieval_simple_wikipedia.py' with 'amberoad/bert-multilingual-passage-reranking-msmarco' as the cross encoder model and it produces an error in the cross_scores

`cross_scores = cross_encoder.predict(cross_inp)`

As an example, I have used two passages and two cross encoders. The passages are:

`passages = ['This is python. It is a snake that is found in the junge', 'Rick n Morty is a cartoon. It is one of the best things ever']`
and the cross encoders are 

`cross_encoder_1 = CrossEncoder('cross-encoder/ms-marco-TinyBERT-L-6')`
`cross_encoder_2 = CrossEncoder('amberoad/bert-multilingual-passage-reranking-msmarco')`

For cross_encoder_1, it produces the following result for the query 'What is python?'

![image](https://user-images.githubusercontent.com/57942901/106448948-bda2f980-6483-11eb-9b68-9c50e2c2d8de.png)

For cross_encoder_2, the result is as follows:

![image](https://user-images.githubusercontent.com/57942901/106449136-edea9800-6483-11eb-8d6f-a5e3b692bc96.png)

This problem is persistent with the original Wikipedia files as well. Could you please suggest a workaround for this case?

**Comment by nreimers** [comment]
cross-encoder/ms-marco-TinyBERT-L-6 produces single scores, while amberoad/bert-multilingual-passage-reranking-msmarco produces two score, with score[1] indicating that score for being relevant.

You could change assignment for the cross-score like this:
= cross_score[idx][1]

**Comment by sbhttchryy** [comment]
Thank you for your super fast response. One quick question.  What does score[0] represent?

**Comment by nreimers** [comment]
> Thank you for your super fast response. One quick question. What does score[0] represent?

The logits that the passage is not relevant for the query.
